engine-runner-bin/Cargo.toml

```diff
@@ -25,13 +25,17 @@ assets = [
     # manually.
     [
         "target/release/libchainflip_engine_v1_4_1.so",
+        # This is the path where the engine dylib is searched for on linux.
+        # As set in the build.rs file.
         "usr/lib/chainflip-engine/libchainflip_engine_v1_4_1.so",
```

Comment:
is it correct that this is a relative path and doesn't start with a slash?

Reply:
this is just how the cargo-deb library does it, agree it's a little strange